frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -110,16 +109,36 @@
 import ButtonCountdown, { countdownStates } from '../ButtonCountdown'
 import Sign from './Sign.vue'
 import L from '../../utils/translations'
-import { votingType } from '../../utils/validators'
+import { votingType, votesObj } from '../../utils/validators'
 import template from 'string-template'
+import { HashableGroupProposal } from '../../../../shared/events'
 
 export default {
   name: 'Voting',
   props: {
     type: {
-      validator: votingType
+      validator: votingType,
+      required: true
     },
-    proposal: Object,
+    votes: {
+      validator: votesObj,
+      required: true
+    },
+    value: {
+      type: [String, Number], // string for add member, remove member, number for rule, mincome
+      required: true
+    },
+    originalValue: Number, // for rule, mincome proposals
+    ownVote: Boolean,
+    isOwnProposal: {
```

Comment:
If we have `isOwnProposal`, maybe `ownVote` should be `isOwnVote` too.

Reply:
I almost wrote "yes of course", but this here is a little trickier. ownVote is not that kind of a boolean, it doesn't say **if this proposal _is_ an own vote**, but it contains the user's own vote that can be 3 things: for (true), against (false), or not voted yet (null). but this raises the question if this could be some other type, not a boolean, to make this less confusing?

Reply:
You're totally right, I was the one who created that prop and didn't remember 😅 , so validation should be `ownVote: [Boolean, null]` is that possible? And add a comment explaining it:
> // can be 3 things: for (true), against (false) or not voted yet (null).